### PR TITLE
SIM-8344 fix

### PR DESCRIPTION
**Pull request by gwaldron**

### CODE REVIEW

File: Examples/ArticulatedModel/src/ExampleArticulatedModel.cpp [hunk 1]
```diff
@@ -153,21 +153,37 @@ class FindNodeByName : public osg::NodeVisitor
 class TankNode
 {
 public:
-  explicit TankNode(simVis::EntityNode* entity)
+  explicit TankNode()
```
Comment (mk401) [inline Examples/ArticulatedModel/src/ExampleArticulatedModel.cpp]: Should no longer be explicit

File: Examples/ArticulatedModel/src/ExampleArticulatedModel.cpp [hunk 2]
```diff
@@ -223,6 +239,50 @@ class TankNode
...
 
+/** Application object that syncs the UI with the Tank data after async model load */
+struct App : public osg::Node
+{
+  TankNode tank_;
+  osg::observer_ptr<osg::Node> node_;
+  osg::ref_ptr<HSliderControl> turretSlider_;
+  osg::ref_ptr<HSliderControl> gunSlider_;
+
+  void refresh();
```
Comment (mk401) [inline Examples/ArticulatedModel/src/ExampleArticulatedModel.cpp]: looks unused

File: Examples/ArticulatedModel/src/ExampleArticulatedModel.cpp [hunk 3]
```diff
@@ -121,7 +121,7 @@ simData::ObjectId addPlatform(simData::DataStore& dataStore, double lat, double
 class FindNodeByName : public osg::NodeVisitor
 {
 public:
-  explicit FindNodeByName(const std::string& name)
+  FindNodeByName(const std::string& name)
```
Comment (mk401) [inline Examples/ArticulatedModel/src/ExampleArticulatedModel.cpp]: well this (and 3 others you updated) should still be explicit..

Comment (gwaldron) [reply Examples/ArticulatedModel/src/ExampleArticulatedModel.cpp]: why?

Comment (mk401) [reply Examples/ArticulatedModel/src/ExampleArticulatedModel.cpp]: It's our code style w.r.t. single parameter constructors, unless there's a good reason for a constructor to not be explicit. Is there a benefit to a non-explicit constructor on these four classes?

Comment (gwaldron) [reply Examples/ArticulatedModel/src/ExampleArticulatedModel.cpp]: Brevity. But I will put it back; did not realize it was your code style.

**Pull request merged**